### DISCUSSION TITLE
Switch Workbench After Loading option has no effect

### DISCUSSION THREAD

**Discussion by cemox** [FreeCAD/FreeCAD#11577] [Discussions that will turn in to issues but need refining]
Changing **"Switch Workbench After Loading"** option in the preferences seems to no effect (neither does **"Close start page after loading option"**. [Ubuntu LTS 22.04]

Is this a bug or am I missing something?

**Comment by luzpaz** [comment]
Full [about](https://wiki.freecad.org/About) info, please

**Comment by cemox** [comment]
```

OS: Ubuntu 22.04.3 LTS (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 0.21.1.33694 (Git) AppImage
Build type: Release
Branch: (HEAD detached at 0.21.1)
Hash: f6708547a9bb3f71a4aaade12109f511a72c207c
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: English/United States (en_US)
Installed mods: 
  * Manipulator 1.5.0
  * Assembly4 0.50.4
  * A2plus 0.4.61
  * 3D_Printing_Tools
  * fasteners 0.4.71
```

**Comment by yorikvanhavre** [comment]
Both work for me... Could you give a step by step of what you are doing?

**Reply by cemox** [reply]
My mistake. It works as soon as I start a new file or load an existing file.  Sorry
